{"name": "Ingresar una letra incorrecta -- @1.2 ", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.SessionNotCreatedException: Message: session not created: Chrome failed to start: exited normally.\n  (session not created: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x55caa3dd55e3 <unknown>\n#1 0x55caa3a980b7 <unknown>\n#2 0x55caa3acee55 <unknown>\n#3 0x55caa3acbb81 <unknown>\n#4 0x55caa3b1647f <unknown>\n#5 0x55caa3b0ccc3 <unknown>\n#6 0x55caa3ad80e4 <unknown>\n#7 0x55caa3ad90ae <unknown>\n#8 0x55caa3d9bce1 <unknown>\n#9 0x55caa3d9fb7e <unknown>\n#10 0x55caa3d894b5 <unknown>\n#11 0x55caa3da07d6 <unknown>\n#12 0x55caa3d6cdbf <unknown>\n#13 0x55caa3dc3748 <unknown>\n#14 0x55caa3dc3917 <unknown>\n#15 0x55caa3dd4773 <unknown>\n#16 0x7f322cc94ac3 <unknown>\n\n", "trace": "  File \"/opt/hostedtoolcache/Python/3.11.7/x64/lib/python3.11/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/opt/hostedtoolcache/Python/3.11.7/x64/lib/python3.11/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"UI-TESTS/steps/ahorcadosteps.py\", line 57, in empieza_juego\n    context.driver = webdriver.Chrome()\n                     ^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.7/x64/lib/python3.11/site-packages/selenium/webdriver/chrome/webdriver.py\", line 45, in __init__\n    super().__init__(\n  File \"/opt/hostedtoolcache/Python/3.11.7/x64/lib/python3.11/site-packages/selenium/webdriver/chromium/webdriver.py\", line 56, in __init__\n    super().__init__(\n  File \"/opt/hostedtoolcache/Python/3.11.7/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 208, in __init__\n    self.start_session(capabilities)\n  File \"/opt/hostedtoolcache/Python/3.11.7/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 292, in start_session\n    response = self.execute(Command.NEW_SESSION, caps)[\"value\"]\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.7/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 347, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.11.7/x64/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Given El jugador empieza el juego", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.SessionNotCreatedException: Message: session not created: Chrome failed to start: exited normally.\n  (session not created: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x55caa3dd55e3 <unknown>\n#1 0x55caa3a980b7 <unknown>\n#2 0x55caa3acee55 <unknown>\n#3 0x55caa3acbb81 <unknown>\n#4 0x55caa3b1647f <unknown>\n#5 0x55caa3b0ccc3 <unknown>\n#6 0x55caa3ad80e4 <unknown>\n#7 0x55caa3ad90ae <unknown>\n#8 0x55caa3d9bce1 <unknown>\n#9 0x55caa3d9fb7e <unknown>\n#10 0x55caa3d894b5 <unknown>\n#11 0x55caa3da07d6 <unknown>\n#12 0x55caa3d6cdbf <unknown>\n#13 0x55caa3dc3748 <unknown>\n#14 0x55caa3dc3917 <unknown>\n#15 0x55caa3dd4773 <unknown>\n#16 0x7f322cc94ac3 <unknown>\n\n", "trace": "  File \"/opt/hostedtoolcache/Python/3.11.7/x64/lib/python3.11/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/opt/hostedtoolcache/Python/3.11.7/x64/lib/python3.11/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"UI-TESTS/steps/ahorcadosteps.py\", line 57, in empieza_juego\n    context.driver = webdriver.Chrome()\n                     ^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.7/x64/lib/python3.11/site-packages/selenium/webdriver/chrome/webdriver.py\", line 45, in __init__\n    super().__init__(\n  File \"/opt/hostedtoolcache/Python/3.11.7/x64/lib/python3.11/site-packages/selenium/webdriver/chromium/webdriver.py\", line 56, in __init__\n    super().__init__(\n  File \"/opt/hostedtoolcache/Python/3.11.7/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 208, in __init__\n    self.start_session(capabilities)\n  File \"/opt/hostedtoolcache/Python/3.11.7/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 292, in start_session\n    response = self.execute(Command.NEW_SESSION, caps)[\"value\"]\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.7/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 347, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.11.7/x64/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1702088439603, "stop": 1702088440264}, {"name": "When El jugador ingresa la letra \"x\"", "status": "skipped", "start": 1702088440265, "stop": 1702088440265}, {"name": "Then Pierde una vida", "status": "skipped", "start": 1702088440265, "stop": 1702088440265}, {"name": "And Se muestra como letra incorrecta \"x\"", "status": "skipped", "start": 1702088440265, "stop": 1702088440265}], "parameters": [{"name": "letra", "value": "x"}], "start": 1702088439603, "stop": 1702088440265, "uuid": "0384cc52-4cff-4519-87ef-5b01b8db7de4", "historyId": "e0227a2fd4eda1668d05b15e75fd322c", "fullName": "Hangman Game: Ingresar una letra incorrecta", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Hangman Game"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}